spec/unleash/client_spec.rb

```diff
@@ -300,6 +300,12 @@
       unleash_client.is_enabled?('any_feature5', {}) { nil }
     ).to be false
 
+    expect(
```

Comment:
Can you pull this out to a separate test and add cases where fallback blocks are sent in as well?
If fallback blocks are failing its not up to the sdk to handle that.

If there is a fallback block we should fallback to that.

Reply:
Hmmm, it's going to be hard to make this fail without using the block but yes, I do agree that the correct behaviour here is probably to fallback to the er... fallback. I'll patch it